pkg/mkimage-raw-efi/make-raw

```diff
@@ -214,6 +216,8 @@ do_inventory_win() {
     local NUM_PART=$(( PART_OFFSET + 5 ))
     local PART_TYPE=EBD0A0A2-B9E5-4433-87C0-68B6B72699C7
 
+    PROTECTIVE_MBR_LIST="$PROTECTIVE_MBR_LIST$(( PART_OFFSET + 4 )):$NUM_PART:"
```

Comment:
I can see here information about config partition (PART_OFFSET + 4). Do we need it to be defined inside inventory (and also _win) function?

Reply:
Like I mentioned in my PR description -- I'd rather not. Yes it is a hack, but it is hack that's going to go away ASAP since I'm working on merging conf partition into EFI one under `/conf` folder. Hence hoping this one line of a hack would not be too much of a problem to merge this PR @giggsoff

Reply:
Ah, sorry, miss this part of description. Thank you for pointing me onto it.